### PR TITLE
iSnd.cpp progress

### PR DESCRIPTION
**Pull request by SquareMan**

### CODE REVIEW

File: src/Core/p2/iSnd.cpp [hunk 1]
```diff
@@ -96,28 +168,80 @@
...
+void iSndWaitForDeadSounds()
+{
+    fc = 0;
+    int i = 0;
+    // Can't get 0x8c to get stored in r31
+    while ((i = fc) < 0x8c) //for (int i = 0; (i = fc) < 0x8c; )
+    {
+        i = fc;
+        while (fc < i + 0xe)
+            ;
```
Comment (mattbruv) [inline src/Core/p2/iSnd.cpp:205]: Idk what's going on with this syntax here, did you run clang-format on this file? this looks weird.

Comment (SquareMan) [reply src/Core/p2/iSnd.cpp:205]: yeah it just won't keep the semicolon there. I meant to add a set of empty braces to make it look nicer but I forgot

Comment (SquareMan) [reply src/Core/p2/iSnd.cpp:205]: actually braces make it worse lol
```
while (fc < i + 0xe)
        {
        };
```

Comment (mattbruv) [reply src/Core/p2/iSnd.cpp:205]: It just looked funny to me, it's fine as it is, I just wanted to make sure you ran the formatting on the file.

Comment (SquareMan) [reply src/Core/p2/iSnd.cpp:205]: I have it enabled on save now

**Review by mattbruv — APPROVED** [review]

**Pull request merged**